pontoon/base/models.py

```diff
@@ -25,12 +25,69 @@
 log = logging.getLogger('pontoon')
 
 
+class UserTranslationsManager(models.Manager):
+    """
+    Provides various method to interact with larger sets of translations and their stats for user.
+    """
+
+    def _changed_translations_count(self, *args):
+        """
+        Helper method, returns expression object which allows us to annotate querysets
+        with counts of translations.
+        """
+        translation_query = (
+            ~Q(translation__string=F('translation__entity__string'))
+            & ~Q(translation__string=F('translation__entity__string'))
+        )
+
+        if args:
+            translation_query &= Q(*args)
```

Comment:
I couldn't find where passing `Q` objects into another `Q` object is documented, and it's kinda weird anyway. Might as well just loop over `args`, which should always be a list anyway, even if it's empty.